File: Python/functions/no checking/pronic.py
#Pronic no.
def pronic(n):
    for i in range(1,n+1):
        if n==i*(i+1):
            return True
    return False
n=int(input("Enter the no. = "))
if pronic(n):
    print("Pronic no.")
else:
    print("not Pronic no.")

File: C Programms/No. Cheking In Range/pronic.c
//WAP to print Pronic no. for 1-n
#include<stdio.h>
void main()
{
    int n,r,s,x;
    printf("Enter the range = ");
    scanf("%d",&r);
    printf("Pronic No. from 1 to %d are ...\n",r);
    for(n=1;n<=r;n++)
    {
        for(int i=1;i<=n;i++)
        {
            if(i*(i+1)==n)
            {
                printf("%d ",n);
                break;
            }
        }
    }
}